cigaradvisor/blocks/contest-form/contest-form.js

```diff
@@ -0,0 +1,15 @@
+export default async function decorate(block) {
+  const iframeWrapper = document.createElement('div');
+  iframeWrapper.id = 'vsscript_148868_314195';
+  block.replaceChildren(iframeWrapper);
+  const p = document.createElement('p');
+  function loadScript() {
+    const script = document.createElement('script');
+    script.async = true;
+    script.type = 'text/javascript';
+    script.src = 'https://app.viralsweep.com/vsa-widget-bedfd7-148868.js?sid=148868_314195';
```

Comment:
This should be provided by the block author. instead of hard-coded.

Reply:
fixed